### PR TITLE
Keys controller tests

### PR DESCRIPTION
**Pull request by Amuhar**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -34,7 +34,7 @@
     "test:watch": "jest --watch",
     "test:cov": "jest --coverage",
     "test:debug": "node --inspect-brk -r tsconfig-paths/register -r ts-node/register node_modules/.bin/jest --runInBand",
-    "test:e2e": "jest --config ./test/jest-e2e.json",
+    "test:e2e": "jest --config  jest-e2e.json",
```
Comment (eddort) [inline package.json:37]: unnecessary space

File: src/http/keys/keys.service.ts [hunk 2]
```diff
@@ -42,7 +52,43 @@ export class KeysService {
...
   async getByPubkey(pubkey: string): Promise<KeyListResponse> {
-    return this.getByPubkeys([pubkey]);
+    const { keys, elBlockSnapshot } = await this.entityManager.transactional(
+      async () => {
+        const { stakingModules, elBlockSnapshot } = await this.stakingRouterService.getStakingModulesAndMeta();
+        const collectedKeys: KeyWithModuleAddress[][] = [];
+
+        for (const module of stakingModules) {
+          const moduleInstance = this.stakingRouterService.getStakingRouterModuleImpl(module.type);
+          const fields: KeyWithModuleAddressFieldT[] = [
```
Comment (eddort) [inline src/http/keys/keys.service.ts:62]: Perhaps we should move this to constants?

File: src/http/keys/keys.service.ts [hunk 3]
```diff
@@ -53,7 +99,14 @@ export class KeysService {
 
         for (const module of stakingModules) {
           const moduleInstance = this.stakingRouterService.getStakingRouterModuleImpl(module.type);
-          const fields: KeyField[] = ['key', 'depositSignature', 'operatorIndex', 'used', 'moduleAddress'];
+          // in result will have extra index field, because it is part of compound pkey
+          const fields: KeyWithModuleAddressFieldT[] = [
```
Comment (eddort) [inline src/http/keys/keys.service.ts:103]: Perhaps we should move this to constants?

File: src/http/sr-modules-keys/sr-modules-keys.controller.ts [hunk 4]
```diff
@@ -41,7 +41,7 @@ export class SRModulesKeysController {
         reply.type('application/json').send(jsonStream);
 
         for (const { keysGenerator, module } of keysGeneratorsByModules) {
-          // TODO: does memory consumption increased ?
+          // TODO: does memory consumption is increase ?
           const keysData = { module, keys: [] as any[] };
```
Comment (eddort) [inline src/http/sr-modules-keys/sr-modules-keys.controller.ts:45]: can we add more strict types?

File: src/storage/module.fixture.ts [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+  exitedValidatorsCount: 0,
+  active: true,
+};
+
+export const srModules = [curatedModule, dvtModule];
+
+// [
+//   [
+//     1,
+//     '0xDc64a140Aa3E981100a9becA4E685f962f0cF6C9',
```
Comment (eddort) [inline src/storage/module.fixture.ts:38]: do we still need that comment?

Comment (Amuhar) [reply src/storage/module.fixture.ts]: removed

File: src/http/sr-modules-keys/sr-modules-keys.controller.ts (hunk 4 above)
Comment (Amuhar) [reply src/http/sr-modules-keys/sr-modules-keys.controller.ts]: returned prev solution as here streams doesnt work properly

Comment (Amuhar) [reply src/http/sr-modules-keys/sr-modules-keys.controller.ts]: and added types

File: package.json (hunk 1 above)
Comment (Amuhar) [reply package.json]: fixed

**Review by br3d — APPROVED** [review]

**Review by eddort — APPROVED** [review]

**Review by infloop — APPROVED** [review]

**Pull request merged**